pkg/proxy/ipvs/proxier.go

```diff
@@ -1659,6 +1659,11 @@ func (proxier *Proxier) writeIptablesRules() {
 	// Note that even if we go over 64, it will still be correct - it
 	// is just for efficiency, not correctness.
 	args := make([]string, 64)
+	loAddr := "127.0.0.0/8"
+	if proxier.ipFamily == v1.IPv6Protocol {
+		loAddr = "::1/128"
+	}
+	proxier.natRules.Write("-A", string(kubeNodePortChain), "-s", loAddr, "-j", "RETURN")
```

Comment:
Move this above the "slice comment" since it has nothing to do with it.
Instead add a new comment that explains that for proxy-mode=ipvs localhost can't be used for nodePorts so it should be dismissed early to omit unnecessary processing.

Reply:
Also change `kubeNodePortChain -> kubeServicesChain` (the "improvement" in https://github.com/kubernetes/kubernetes/pull/112559#issuecomment-1259052540). For proxy-mode=ipvs **no** services can use localhost, not only nodePort.

Localhost addresses are never inserted in ipvs (check with ipvsadm) so to let those packets pass all rules in kubeServicesChain (and sub-chains) are just wrong.

Reply:
done, PTAL